#!/usr/bin/env python3
# _*_ coding:utf-8 _*_

PACKAGE = "rm_detector"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("g_nms_thresh",double_t, 1, "g_nms_thresh", 0.1,0.1, 1.0)
gen.add("g_bbox_conf_thresh",double_t, 2, "g_bbox_conf_thresh", 0.1,0.1, 1.0)
gen.add("g_turn_on_image",bool_t,3,"g_turn_on_image",True)


exit(gen.generate(PACKAGE, PACKAGE, "dynamic"))
